release-notes/cabal-install-3.8.1.0.md

```diff
@@ -22,19 +28,41 @@ cabal-install 3.8.1.0 Changelog
+  - add `--enable-append` and `--disable-append` flags to `v2-configure`, which toggle the option for the new configuration to be appended to the old config file
+  - add `--enable-backup` and `--disable-backup` flags to `v2-configure`, which toggle the backup feature for the configuration file
 
-  - The release binaries are now built with GHC 9.2.3, which fixes some minor snags
+- Better support for scripts [#5508](https://github.com/haskell/cabal/issues/5508) [#5698](https://github.com/haskell/cabal/issues/5698) [#6149](https://github.com/haskell/cabal/issues/6149) [#6354](https://github.com/haskell/cabal/issues/6354) [#7073](https://github.com/haskell/cabal/issues/7073) [#7842](https://github.com/haskell/cabal/issues/7842) [#7851](https://github.com/haskell/cabal/pull/7851) [#7925](https://github.com/haskell/cabal/pull/7925) [#7938](https://github.com/haskell/cabal/pull/7938) [#7990](https://github.com/haskell/cabal/pull/7990) [#7997](https://github.com/haskell/cabal/pull/7997)
+
+  - Script support improved or added across relevant commands.
+  - `cabal run script` will now cache results and will not do a fresh build every time.
+  - `cabal build script` added: It will build the cache for script.
+  - `cabal repl script` added: It will open a repl for script using the cache if available.
+  - `cabal clean script` added: It will clean the cache for script.
+  - `cabal clean` will now remove script caches for which there is no marching script.
+  - `cabal list-bin` now works with scripts
+  - The name of the generated script executable has been changed from "script" to
+    "cabal-script-<your-sanitized-script-name>" for easier process management.
+  - Reduce the default verbosity of scripts, so that the build output doesn't interfere with the script output.
+  - Scripts now support a project metadata block that allows them to use options
+    that would normally be set in a cabal.project file.
+
+- `cabal-install-solver`: Backtrack when no pkg-config is present [#7448](https://github.com/haskell/cabal/issues/7448) [#7621](https://github.com/haskell/cabal/pull/7621)
+
+  When solving for pkgconfig-depends, when pkg-config is not present, the cabal solver will now backtrack and try a different automatic flag and dependency configuration, just as it does if pkg-config is present, but does not contain the specified package.
+
+- Apply local options only to local packages [#7998](https://github.com/haskell/cabal/issues/7998) [#7973](https://github.com/haskell/cabal/pull/7973)
+
+  - Command-line `ghc-options` only applies to local packages
+  - `program-options` stanza only applies to local packages
+
+- Add `--open` flag to `cabal haddock` [#7366](https://github.com/haskell/cabal/issues/7366) [#7550](https://github.com/haskell/cabal/pull/7550)
```

Comment:
While these aren't too significant (or are they?), I moved them under this section because:

* the first is likely to affect configuration files
* the second is an often requested feature